mypi.ini

```diff
@@ -21,6 +21,8 @@ disallow_untyped_defs = True
 [mypy-scriptworker.constants]
 check_untyped_defs = True
 disallow_untyped_defs = True
+# untyped_calls because of immutabledict==1.3.0
+disallow_untyped_calls = False
```

Comment:
Nice, thank you!
Looks like we're still failing on `src/scriptworker/constants.py:34: error: Need type annotation for 'DEFAULT_CONFIG'`

Reply:
Bah, thought I pushed that fixup too, amended now.